lib/tasks/test.rake

```diff
@@ -0,0 +1,65 @@
+# encoding: utf-8
+OS_PLATFORM = RbConfig::CONFIG["host_os"]
+VENDOR_PATH = File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "vendor"))
+
+if OS_PLATFORM == "linux"
+  FILEBEAT_URL = "https://beats-nightlies.s3.amazonaws.com/filebeat/filebeat-5.0.0-alpha4-SNAPSHOT-linux-x86_64.tar.gz"
```

Comment:
can you get the logstash version from somewhere and try to download the same filebeat version or is that unfeasible? otherwise you always need to update this one...